src/EditorFunction/MoodChanger.as

```diff
@@ -0,0 +1,39 @@
+namespace EditorHelpers
+{
+    [Setting category="Functions" name="Mood Changer: Enabled" description="Uncheck to disable plugin function for mood changer"]
+    bool Setting_MoodChanger_Enabled = true;
+
+    class MoodChanger : EditorHelpers::EditorFunction
+    {
+        string UserTimeSet = Editor.MoodTimeOfDayStr;
```

Comment:
The plugin is throwing a null pointer exception when the game starts. I believe this is a contributing factor because the editor is not open or accessible when openplanet first loads all plugins.

Reply:
Fixed in e4ea1512c49d78cbb254839a1f2a082ef8f4284e